dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/SchemesAndSchemesContentTypeView.java

```diff
@@ -0,0 +1,30 @@
+package com.dotcms.rest.api.v1.workflow;
+
+import com.dotmarketing.portlets.workflows.model.WorkflowScheme;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * View just to encapsulate all schemes and all schemes for the content type
+ * @author jsanca
+ */
+public class SchemesAndSchemesContentTypeView implements Serializable {
+
+    private final List<WorkflowScheme> schemes;
+    private final List<WorkflowScheme> contentTypeSchemes;
+
+    public SchemesAndSchemesContentTypeView(List<WorkflowScheme> schemes, List<WorkflowScheme> contentTypeSchemes) {
+
+        this.schemes = schemes;
+        this.contentTypeSchemes = contentTypeSchemes;
+    }
+
+    public List<WorkflowScheme> getSchemes() {
+        return schemes;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "schemes". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)